config/settings/__init__.py

```diff
@@ -42,6 +43,9 @@
         'rest_framework.renderers.BrowsableAPIRenderer',
     )
 
+    # use same password hasher as for the rdmo testing fixtures
+    PASSWORD_HASHERS = ("django.contrib.auth.hashers.MD5PasswordHasher",)
```

Comment:
This should not be the default (for DEBUG) since very little people use the user fixtures. It will confuse people when switching on debug while testing RDMO.

Reply:
We could add a setting though, like `DEBUG_TOOLBAR`.